prowler/providers/aws/services/iam/iam_service.py

```diff
@@ -408,6 +409,20 @@ def __list_server_certificates__(self):
         finally:
             return server_certificates
 
+    def __list_tags_for_resource__(self):
```

Comment:
I think is better to split this function into two, one for users and the other for roles.

Reply:
Done